content/security/for-maintainers.adoc

```diff
@@ -123,6 +123,40 @@ This strikes a balance between allowing the longest possible time for reviews an
 The Jenkins security team needs to prepare a Maven staging repository before security updates can be staged, so follow the instructions below only once you know the Maven repository to stage to.
 Make sure to follow instructions provided by the Jenkins security team if they deviate from the instructions below.
 
+[[backporting]]
+=== Backporting
+
+At this point, consider the need for backports.
+Some administrators may be unable to install the latest version of your plugin.
+Usually this is because the plugin depends on a recent weekly release of Jenkins.
+Users of the link:/download/lts/[LTS line of Jenkins releases] will not be offered a plugin update that requires a weekly version of Jenkins higher than the LTS baseline, even when they're already on the latest version of Jenkins available to them.
+
+We recommend that all security fixes are made available to both current weekly and current LTS releases.
+If the current LTS release is the first in its line (e.g., 2.375.1), also consider making the security fix available to users of the previous release, as many administrators may not immediately update.
+
+Additionally, the security fixes should generally be made available to users who are already on the latest version of the plugin available to them, rather than an earlier version.
+This will make it easy to update to a version with the fix through the plugin manager, without potentially losing features by doing what amounts to a manual downgrade.
+
```

Comment:
Would it be possible to add a bit more detail in here (or links to other pages) about the different versioning systems used in Jenkins plugins? (The CD and manual versioning systems)